process/smartContract/process.go

```diff
@@ -2745,6 +2752,34 @@ func (sc *scProcessor) checkUpgradePermission(contract state.UserAccountHandler,
 	return process.ErrUpgradeNotAllowed
 }
 
+func (sc *scProcessor) createCompleteEventLogIfNoMoreAction(
+	tx data.TransactionHandler,
+	txHash []byte,
+	results []data.TransactionHandler,
+) *vmcommon.LogEntry {
+	for _, scr := range results {
+		dstShardID := sc.shardCoordinator.ComputeId(scr.GetRcvAddr())
+		isCrossShard := sc.shardCoordinator.SelfId() != dstShardID
+		if isCrossShard && !sc.isInformativeSCR(scr) {
```

Comment:
so we stop and say: the call is not final yet if we find a cross shard & not informative SCR

Reply:
exactly.